bip-newproposal.mediawiki

```diff
@@ -0,0 +1,140 @@
+<pre>
+  BIP: TBD
+  Title: PQ-Derived Schnorr: Hybrid Post-Quantum and Schnorr Signatures
+  Author: Author: [Caleb Lee] director@j1729labs.online, [Justin Park] justin7361@j1729labs.online, [Eunice Lee] cuspro0103@j1729labs.online, [Sophia Shim] zypo1015@j1729labs.online
```

Comment:
[“If there are multiple authors, each should be on a separate line following RFC 2822 continuation line conventions.”](https://github.com/bitcoin/bips/blob/master/bip-0002.mediawiki#user-content-BIP_header_preamble)

```suggestion
  Author: Caleb Lee <director@j1729labs.online>
          Justin Park <justin7361@j1729labs.online>
          Eunice Lee <cuspro0103@j1729labs.online>
          Sophia Shim <zypo1015@j1729labs.online>
```